ts/Series/DependencyWheel/DependencyWheelSeries.ts

```diff
@@ -254,8 +262,8 @@ class DependencyWheelSeries extends SankeySeries {
      * functions instead of the whole translate function.
      */
     public translate(): void {
-
-        const options = this.options,
+        const dependecyWheel = this,
```

Comment:
Why do we need a local reference? And if we really need one, the convention is to use `series` in this case.